src/app/lifecycle.ts

```diff
@@ -119,16 +169,80 @@ export class AppStore {
   }
 }
 
+/**
+ * Checks to see if the provided appName is a non-empty string and throws if it
+ * is not.
+ * 
+ * @param appName A string representation of an App name.
+ * 
+ * @throws FirebaseAppError if appName is not of type string or is empty.
+ * 
+ * @internal
+ */
+function validateAppNameFormat(appName: string): void {
+  if (typeof appName !== 'string' || appName === '') {
```

Comment:
Can we use `validator.isNonEmptyString()` here?
https://github.com/firebase/firebase-admin-node/blob/26b884f70300ef8e3213dfbbd72351ce7f68c695/src/utils/validator.ts#L104